src/schemas/listing-core.json

```diff
@@ -86,15 +85,15 @@
     "unitListing": {
       "type": "object",
       "properties": {
-        "unitsAvailable": {
+        "unitsForSale": {
```

Comment:
Why this? Not everything is "for sale", per se. 🤔

Reply:
There was code added recently to compute in origin-js "unitsAvailable" based on "unitsForSale" and "unitsSold". See https://github.com/OriginProtocol/origin-js/blob/8c1b19abd33fad514dd09ad6a6145ec52ce6818c/src/resources/marketplace.js#L88

So I just sticked to that convention... Happy to change it if it is cleaner.

Reply:
Oh, right. I forgot about that distinction. So "available" is more like "remaining" and "forSale" is more like the total supply? 😕

Reply:
Yes, you are correct. I'm pretty bad at naming things though so happy to make changes...
Do you think this would be more clear ?
 - unitsTotal (field in the schema): total number of units available
 - unitsRemaining (computed by origin-js. difference of unitsTotal-unitsSold): number of units remaining
 - unitsSold (computed by origin-js based on offer data)
Though to your earlier point everything is not for sale so "unitsSold" is not great.... hmmm any better idea ?

Reply:
I think that is definitely an improvement. 👍

Reply:
Ok. I'm changing things to unitsTotal, unitsRemaining, unitsSold